projects/valtimo/dossier-management/src/lib/dossier-management-detail-container/dossier-management-detail-container.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="main-content">
+  <div class="container-fluid">
+    <div class="col-12 px-0 mb-5">
+      <valtimo-widget>
+        <ul class="nav nav-tabs">
+          <li class="nav-item">
+            <a id="test-tab" class="nav-link clickable active" data-toggle="tab" (click)="displayBodyComponent('case')">
+              {{ 'dossierManagement.tabs.cases' | translate }}
+            </a>
+          </li>
+          <li class="nav-item">
+            <a id="test-tab-1" class="nav-link clickable" data-toggle="tab" (click)="displayBodyComponent('search')">
+              {{ 'dossierManagement.tabs.searchFields' | translate }}
+            </a>
+          </li>
+        </ul>
+
+        <div class="card-body bg-white p-5 position-relative tab-container">
+
+           <div *ngIf="isCase">
+            <valtimo-dossier-management-tab-details></valtimo-dossier-management-tab-details>
+          </div>
+
+          <div *ngIf="!isCase">
+            <p>search fields</p>
```

Comment:
Should be your search field component?

Reply:
added the new component <valtimo-dossier-management-search-fields>